charts/business-api-ecosystem/values.yaml

```diff
@@ -58,15 +58,26 @@ siop:
   ccs:
     # -- Endpoint of the CCS
     endpoint: "http://credentials-config-service:8080"
-    # -- Credential configuration to be registered
-    credentials:
-      - type: "VerifiableCredential"
-        trustedParticipantsLists: [
-          "https://tir.fiware.dev"
-        ]
-        trustedIssuersLists: [
-          "https://til.fiware.dev"
-        ]
+    # -- Default scope to be used from scopes below, if none is provided
+    defaultScope: "defaultScope"
+    # -- Credential configurations for particular scopes
+    scopes:
+      defaultScope:
+        - type: "ParticipantRegistryDefaultCredential"
+          trustedParticipantsLists: [
+            "https://tir.fiware.dev"
+          ]
+          trustedIssuersLists: [
+            "https://til.fiware.dev"
+          ]
+      didRead:
```

Comment:
Do we need this one? TIR DIDRegistry is not used and the TIR API doesn't care for this scope

Reply:
It's just meant to have an example on how to fill this scopes object. In the end it's up to the final config what to put there.